core/core-api/src/main/java/com/hbc/pms/core/api/config/WebSocketConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.hbc.pms.core.api.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.simp.config.MessageBrokerRegistry;
+import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker;
+import org.springframework.web.socket.config.annotation.StompEndpointRegistry;
+import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer;
+
+@Configuration
+@EnableWebSocketMessageBroker
+public class WebSocketConfig implements WebSocketMessageBrokerConfigurer {
+    public static final String LOCAL_ORIGIN = "http://localhost:5173";
```

Comment:
hardcode

Reply:
I've change the origins to be read from application.yml